pages/dashboard/ServiceInvitations.js

```diff
@@ -0,0 +1,38 @@
+import _ from 'lodash';
+import getConfig from 'next/config';
+
+import DisplayInvitations from './DisplayInvitations';
+
+/**
+ * COMPONENT 'ApplicationSection'
+ *
+ * @TODO
+ *
+ *
+ * @param {String} name — name of the Application
+ * @param {String} id — id of the application
```

Comment:
This is not accurate... I can see many more parameters down there.
Also, what is the "id of the application"?